bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -6382,6 +6382,28 @@ spec:
                 default: netobserv
                 description: Namespace where NetObserv pods are deployed.
                 type: string
+              networkPolicy:
+                description: '`networkPolicy` defines network policy settings for
```

Comment:
```suggestion
                description: '`networkPolicy` defines ingress network policy settings for
```

Reply:
sure, I need to change the go file & regenerate the bundle